src/index.html

```diff
@@ -4,8 +4,8 @@
   Simplefolio is a clean and responsive portfolio template for Developers!
   Created by Jacobo Martinez.
   -
-  Github Repo: https://github.com/cobidev/simplefolio/
-  Readme: https://github.com/cobidev/simplefolio/README.md
+  Github Repo: https://github.com/cobiwave/simplefolio/
+  Readme: https://github.com/cobiwave/simplefolio/README.md
```

Comment:
readme link should rather be https://github.com/cobiwave/simplefolio/blob/master/README.md